src/Markdown.MAML/packages.config

```diff
@@ -0,0 +1,4 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="YamlDotNet" version="4.2.1" targetFramework="net45" />
+</packages>
```

Comment:
This is nuget config, right?
Can you, consolidate it with `.nuget/*` files?
I don't know much about modern way of managing dependencies with nuget. We have this checked-in `nuget.exe` which doesn't seem like a good idea.
Maybe we can use adding new dependency as an oportunity to streamline this?